evm/contracts/ChainlinkClient.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.4.24;
+
+import "./Chainlinked.sol";
+
+/**
+ * @title The ChainlinkClient contract
+ * @notice Contract writers can inherit this contract in order to create requests for the
+ * Chainlink network. ChainlinkClient is an alias of the Chainlinked contract.
+ */
+contract ChainlinkClient is Chainlinked {
```

Comment:
Does this cost any extra gas? This is not quite an alias, there may be some dynamic dispatch penalty depending on how solidity implements virtual calls.

Reply:
It costs a bit more to deploy (64 gas by my spot check), but once deployed it does not affect function calls, like creating requests.

Reply:
Now with functions implemented it's 3152 more gas to deploy and 49 more gas to use the updated function names (since they simply call the old function names).

Reply:
We could flip and have the old names aliased to the new ones for backwards compatibility.